src/webhint-theme/helper/utils.js

```diff
@@ -154,7 +154,7 @@ module.exports = {
         // If the markdown Content should be used.
         const guildeIndexes = ['contributor guide', 'user guide'];
```

Comment:
Was this variable meant to be named `guideIndexes` (instead of `guildeIndexes`)? I know it wasn't introduced by this PR, but would be nice to fix the typo since we're in the area.

Reply:
No idea :s